File: Obligatorio1DA2/Domain/Lodging.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain
{
    public class Lodging
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public TouristicPoint TouristicPoint{get; set;}
        public string Description { get; set; }
        public string Direction { get; set; }
        public string Phone { get; set; }
        public int Stars { get; set; }
        public double Price { get; set; }
        public string Images { get; set; }
        public bool IsDeleted { get; set; }
        public bool Capacity { get; set; }


        public Lodging()
        {

        }

        public override bool Equals(object obj)
        {
            if (obj == null)
            {
                return false;
            }
            if (!(obj is Lodging))
            {
                return false;
            }
            return Name == ((Lodging)obj).Name 
                && Direction == ((Lodging)obj).Direction;
        }
    }
}

File: Codigo/UruguayNatural/src/Models/Lodging.ts
import { TPoint } from './TPoint';

export class Lodging {
    Id: number;
    Name: string;
    TouristicPoint: TPoint;
    Description: string;
    Direction: string;
    Phone: string;
    Stars: number;
    Price: number;
    Images: string[];
    Score: number;
    Capacity: boolean;

    constructor(Id: number, Name: string, TouristicPoint: TPoint, Description: string, Direction: string,
                Phone: string, Stars: number, Price: number, Images: string[], Score: number, Capacity: boolean) {
      this.Id = Id;
      this.Name = Name;
      this.Description = Description;
      this.Direction = Direction;
      this.Phone = Phone;
      this.Images = Images;
      this.Price = Price;
      this.Score = Score;
      this.Stars = Stars;
      this.TouristicPoint = TouristicPoint;
      this.Capacity = Capacity;

  }

}
